docs/antora.yml

```diff
@@ -30,4 +28,4 @@ nav:
 # On release, update this to the appropriate released camel version.
 asciidoc:
   attributes:
-    camel-version: next
+    camel-version: 3.12.x
```

Comment:
Is this correct, or should it be 3.13.x? The original plan was to remove the 3.12.x docs a couple days ago, until we found out that the current camel-quarkus released version actually depends on 3.12.x.  If 3.12.x is correct for camel-k, it will mean we keep the camel 3.12.x docs around until the next release of camel-k which will presumably depend on a later version of camel. This is totally fine and appropriate if it accurately describes the dependencies between the projects.
I also wonder if 'next' is the appropriate value in the camel-k main branch.
I would really like a clear description in the docs somewhere of how each subproject version relates to a main camel version.

Reply:
> Is this correct, or should it be 3.13.x? The original plan was to remove the 3.12.x docs a couple days ago, until we found out that the current camel-quarkus released version actually depends on 3.12.x. If 3.12.x is correct for camel-k, it will mean we keep the camel 3.12.x docs around until the next release of camel-k which will presumably depend on a later version of camel. This is totally fine and appropriate if it accurately describes the dependencies between the projects.

I think it is 3.12.x, as Camel K also depends on that Camel Quarkus version:

https://github.com/apache/camel-k-runtime/blob/eb37ecbe7a829f4a09dc5e9f5ebc8b840d936f0b/pom.xml#L41

> I also wonder if 'next' is the appropriate value in the camel-k main branch.

I think that's appropriate AFAIU.

Reply:
There may be a more complicated network of dependencies, that might lead to an inconsistent result, but if you follow the path from camel-k 'next' to camel-quarkus 'next' to camel, you will end up  currently, I think, at 3.13.x.  I can't think of an automated way of syncing these versions across projects.

Reply:
Right. At some point, the `next` versions have to be changed to the actual versions, but I'm not sure that decision can be automated, or if there is a better process.